sql/hive/src/test/scala/org/apache/spark/sql/hive/client/HiveClientSuite.scala

```diff
@@ -316,6 +335,17 @@ class HiveClientSuite(version: String) extends HiveVersionSuite(version) {
     assert(!client.tableExists("temporary", "tgt"))
   }
 
+  test("alterTable - change collation") {
```

Comment:
is it the same with the test `create/alter table with collations`?

Reply:
Right, this test case covers a subset of that one (just the alter table default collation part), so I have removed this test case.